notebooks/README.md

```diff
@@ -1,24 +1,49 @@
-Somewhat experimental setup for tracking canonical versions of Learn notebooks in version control.
+The 'notebook pipeline' in this directory is a pseudo templating system with kaggle kernels API integration.
 
-`partials/python/` contains 'templates' for all the notebooks in the Python track (with `tut_1.ipynb` being the first tutorial notebook, `ex_1.ipynb` being the corresponding exercise notebook, etc.). This is where notebook editing should happen.
+## Directory structure
 
-These notebooks contain some 'macros' in the source of some cells, such as `#$END_OF_EXERCISE$` or `#$EXERCISE_URL(4)$`. These get replaced (see `lesson_preprocessor.py`) when rendering the final notebook. To render some notebooks (writing them to the `rendered/` subdirectory), use `render.sh`. e.g.
+To start a new track (i.e. a themed sequence of notebooks such as are found on the Kaggle Learn homepage, or in Kaggle Learn Challenges) called "spam", run `new_track.sh spam`. This will create a `spam/` subdirectory, with the following contents:
 
-    render.sh partials/python/*.ipynb
+```
+partials/
+rendered/
+pushables/
+__init__.py
+track_meta.py
+track_config.yaml
+```
 
-When you're ready to sync changes to the site, run `prepare_push.py`, passing the name of the track to prepare (e.g. `prepare_push.py python`). This will create a directory structure in a subdirectory `pushables` which is amenable to the Kernels API.
+`partials/` is where the notebooks you author should go. These will be straightforward ipynb notebooks which (perhaps with some setup such as downloading necessary datasets, installing libraries, and careful path manipulation) may be runnable locally. However, they're *also* recipes for generating ipynb notebooks.
 
-To push a particular notebook, do something like...
+`rendered/` is where the notebooks generated from the `partials/` recipes go. These are what get synced to Kaggle Kernels. You should not edit these directly (treat them as build products).
 
-    cd pushables/python/ex_4
-    kaggle k push -p .
+`pushables/` contains `kernel-metadata.json` files for syncing notebooks to Kaggle Kernels via the API.
```

Comment:
I'd favor renaming this from `pushables` to `kernels_metadata`. 

`pushables` made me think it would be the content that gets pushed.

Reply:
It used to be! There used to be a limitation in the API where the metadata file and the .ipynb file to be pushed had to be in the same directory. This name is kind of a relic. Agreed, kernels_metadata would be a better name.